lang/cs/Org.Apache.REEF.IMRU.Examples/MapperCount/IdentityMapFunction.cs

```diff
@@ -26,7 +28,7 @@ namespace Org.Apache.REEF.IMRU.Examples.MapperCount
     public sealed class IdentityMapFunction : IMapFunction<int, int>
     {
         [Inject]
-        private IdentityMapFunction()
+        private IdentityMapFunction(IInputPartition<Stream> dataPartition)
```

Comment:
@afchung  So partition configuration is now passed only in context and not task. So it ensures that map function has access to partition. But you are right that it does not check whether partition is created twice which would be bug in REEF then.  Do you want me to check that? For this I have to then create another Dataset with static counter in partition constructor which throws error if count exceeds one. Let me know if you want me to introduce it.